data.sql

```diff
@@ -7,3 +7,57 @@ VALUES
 ('Pikachu', '07-01-2021', 1, false, 15.04),
 ('Devimon', '12-05-2017', 5, true, 11)
 ;
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '08-02-2020', 0, false, 11),
+('Plantmon', '15-11-2021', 2, true, 5.7),
+('Squirtle', '02-04-1993', 3, false, 12.13),
+('Angemon', '12-06-2005', 1, true, 45),
+('Boarmon', '07-06-2005', 7, true, 20.4),
+('Blossom', '13-10-1998', 3, true, 17),
```

Comment:
- Hey, Please, let's make the value for the weight a floating number and not an integer here. `17.0`